packages/jest-jasmine2/src/jasmineAsyncInstall.js

```diff
@@ -125,21 +125,73 @@ function promisifyIt(originalFn, env, jasmine) {
   };
 }
 
-function makeConcurrent(originalFn: Function, env) {
+function makeQueue(max: number) {
```

Comment:
could we use `throat` for this (same thing we use for workers?). https://yarnpkg.com/en/package/throat